lib/ReactViews/Analytics/InvokeFunction.jsx

```diff
@@ -32,6 +32,10 @@ const InvokeFunction = React.createClass({
         this.initializeParameters(nextProps);
     },
 
+    componentWillUpdate(nextProps, nextState) {
+        this.initializeParameters(nextProps);
+    },
+
```

Comment:
Hmm... InvokeFunction does some interesting contortions, doesn't it? 
Eg. It seems strange to `knockout.track` the same thing in both `componentWillMount` and `initializeParameters`. And that `initializeParameters` also "enable[s] the context item, if any" - can that be moved somewhere more appropriate?
But regarding this PR, I'm worried that the `initializeParameters` will overwrite desirable parameters with the default values in some situation we haven't come across yet.
Perhaps you need a function "updateParameters", which updates the parameters using the default values just as - er - default values?